exercises/05-reselect/cypress/integration/user_list_spec.ts

```diff
@@ -0,0 +1,48 @@
+import { UserList } from '../support/constants'
+
+describe('The Home Page', () => {
+  it('successfully loads', () => {
+    cy.visit('/')
+    cy.contains('h1', UserList.header, { matchCase: false })
+
+    cy.contains('button', UserList.addAryaButton, { matchCase: false })
+    cy.contains('button', UserList.addDaenerysButton, { matchCase: false })
+
+    cy.contains('thead', 'First Name', { matchCase: false }).contains(
+      'thead',
+      'Last Name',
+      {
+        matchCase: false,
+      }
+    )
+    cy.contains('tbody', 'No Users')
+  })
+
+  it('adds users', () => {
+    cy.visit('/')
+
+    // add multiple users
```

Comment:
Not necessarily something that needs to be changed.

But if you turn these "blocks" into functions with well described names (these functions may eventually be scoped here so that they are explicitly private if they are meant to be), you would not need to use the comments and also managed to split the function a little bit - not that it would be required.

Reply:
Reason i am mentioning this is that i always prefer good naming & design over comments as you typically don't need to maintain that (unlike comments) and if you do, then you will think twice if something can or should be changed.

Reply:
I agree with the maintainability concern of the comments. In my previous workplace we had a style guide which forced us to write a large amount of comments. They can so easily become outdated without anyone noticing. 

Nevertheless, I find them very useful for educating purposes: in examples and tutorials. They help to understand the purpose of code more quickly and act as an additional reassurance that the code was understood correctly.

Reply:
Fine by me.